### DISCUSSION TITLE
print to lcd

### DISCUSSION THREAD

**Discussion by kurt48kr** [forntoh/LcdMenu#181] [Q&A] [answered]
hi there
I cannot figure out how to print to the lcd  which a LiquidCrystalI2CAdapter lcdAdapter(0x3f, LCD_COLS, LCD_ROWS);
 menu.hide();
   //lcdAdapter
   menu.setCursorPosition(0);
is working
but the old code 
menu.hide();
menu.lcd->setCursor(0, 0);
menu.lcd->print("hh:mm:ss");

does not work

kind regards
Kurt K

**Comment by forntoh** [comment, marked as answer]
The menu no longer has direct reference to the lcd, it's now in the adapter.
So do Use `lcdAdapter.lcd` to get the instance of the `lcd`

**Reply by kurt48kr** [reply]
hi again
thank you for the answer, that works.
is there more things that differs from the old versions?

kind regards
kurt

**Reply by forntoh** [reply]
https://github.com/forntoh/LcdMenu/releases/tag/4.0.0

**Reply by kurt48kr** [reply]
thank you.
kind regards
kurt
